layouts/partials/footer.html

```diff
@@ -1,8 +1,20 @@
 </main>
     <footer>
-        <h6>{{ .Site.Copyright | markdownify }} |
-            {{ T "rendered" }} <a href="https://gohugo.io" title="Hugo">Hugo</a> |
-            <a href="{{.Site.BaseURL}}index.xml">{{ T "subscribe" }} </a></h6>
+        <ul>
+            <li>
+                <h6>{{ .Site.Copyright | markdownify }} |
```

Comment:
This will cause conflicts with the other PR. I will fix this once one of the is merged.

Reply:
Thanks. I've committed the first PR. So you can update this PR.

Reply:
Done, thanks! :pray: